libwild/src/elf_writer.rs

```diff
@@ -244,6 +245,7 @@ fn fill_padding(mut section_buffers: OutputSectionMap<&mut [u8]>) {
     });
 }
 
+#[tracing::instrument(skip_all, name = "Sort .eh_frame_hdr")]
```

Comment:
This line was removed in a separate PR. It's replaced by the `timing_phase!` line inside the function.